airbyte-server/build.gradle

```diff
@@ -1,5 +1,28 @@
 plugins {
     id 'application'
+    id 'maven-publish'
+    id 'com.github.johnrengelman.shadow' version '6.1.0'
+}
+
+shadowJar {
+    zip64 true
+    mergeServiceFiles()
+    exclude 'META-INF/*.RSA'
+    exclude 'META-INF/*.SF'
+    exclude 'META-INF/*.DSA'
+    // Not stubbing this out adds 'all' to the end of the jar's name.
```

Comment:
@jrhizor I had to add this so the names of the jars would line up. We either stub this out or we change the Cloud's dependency to be {module]-all.jar.

Reply:
Hmm it was able to consume the `-all` jar for me properly for me. Not sure what made it required for you but it's fine either way.

Reply:
Oh this is for the server Docker build?